bug: Failed to load native binding

As reported in https://github.com/apify/camoufox-js/issues/9

There are some differences between the user's stack and our dev / testing stack:
- Windows
- pnpm
- Installed as dependency of `camoufox-js`. Could this cause any issues?

```log
:\Codes\nodejs\tests\browser-test>pnpm start

> ts-pkg-template@1.0.0 start C:\Codes\nodejs\tests\browser-test
> npx tsx src/index.ts

C:\Codes\nodejs\tests\browser-test\node_modules\.pnpm\impit@0.2.5\node_modules\impit\index.js:362
    throw new Error('Failed to load native binding', { cause: loadErrors })
          ^

Error: Failed to load native binding
    at Object.<anonymous> (C:\Codes\nodejs\tests\browser-test\node_modules\.pnpm\impit@0.2.5\node_modules\impit\index.js:362:11)
    at Module._compile (node:internal/modules/cjs/loader:1554:14)
    at Object.transformer (C:\Users\Administrator\AppData\Local\npm-cache\_npx\fd45a72a545557e9\node_modules\tsx\dist\register-DCnOAxY2.cjs:2:1186)
    at Module.load (node:internal/modules/cjs/loader:1289:32)
    at Function._load (node:internal/modules/cjs/loader:1108:12)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
    at Module.require (node:internal/modules/cjs/loader:1311:12)
    at require (node:internal/modules/helpers:136:16)
    at Object.<anonymous> (C:\Codes\nodejs\tests\browser-test\node_modules\.pnpm\impit@0.2.5\node_modules\impit\index.wrapper.js:1:16) {
  [cause]: [
    Error: Cannot find module './impit-node.win32-x64-msvc.node'
    Require stack:
    - C:\Codes\nodejs\tests\browser-test\node_modules\.pnpm\impit@0.2.5\node_modules\impit\index.js
        at node:internal/modules/cjs/loader:1225:15
        at nextResolveSimple (C:\Users\Administrator\AppData\Local\npm-cache\_npx\fd45a72a545557e9\node_modules\tsx\dist\register-DCnOAxY2.cjs:3:942)
        at C:\Users\Administrator\AppData\Local\npm-cache\_npx\fd45a72a545557e9\node_modules\tsx\dist\register-DCnOAxY2.cjs:2:2550
        at C:\Users\Administrator\AppData\Local\npm-cache\_npx\fd45a72a545557e9\node_modules\tsx\dist\register-DCnOAxY2.cjs:2:1624
        at resolveTsPaths (C:\Users\Administrator\AppData\Local\npm-cache\_npx\fd45a72a545557e9\node_modules\tsx\dist\register-DCnOAxY2.cjs:3:760)
        at C:\Users\Administrator\AppData\Local\npm-cache\_npx\fd45a72a545557e9\node_modules\tsx\dist\register-DCnOAxY2.cjs:3:1038
        at m._resolveFilename (file:///C:/Users/Administrator/AppData/Local/npm-cache/_npx/fd45a72a545557e9/node_modules/tsx/dist/register-RyGUjI6j.mjs:1:789)
        at Function._load (node:internal/modules/cjs/loader:1055:27)
        at TracingChannel.traceSync (node:diagnostics_channel:322:14)
        at wrapModuleLoad (node:internal/modules/cjs/loader:220:24) {
      code: 'MODULE_NOT_FOUND',
      requireStack: [
        'C:\\Codes\\nodejs\\tests\\browser-test\\node_modules\\.pnpm\\impit@0.2.5\\node_modules\\impit\\index.js'
      ]
    },
    Error: The specified module could not be found.
    \\?\C:\Codes\nodejs\tests\browser-test\node_modules\.pnpm\impit-win32-x64-msvc@0.2.5\node_modules\impit-win32-x64-msvc\impit-node.win32-x64-msvc.node
        at Object..node (node:internal/modules/cjs/loader:1732:18)
        at Module.load (node:internal/modules/cjs/loader:1289:32)
        at Function._load (node:internal/modules/cjs/loader:1108:12)
        at TracingChannel.traceSync (node:diagnostics_channel:322:14)
        at wrapModuleLoad (node:internal/modules/cjs/loader:220:24)
        at Module.require (node:internal/modules/cjs/loader:1311:12)
        at require (node:internal/modules/helpers:136:16)
        at requireNative (C:\Codes\nodejs\tests\browser-test\node_modules\.pnpm\impit@0.2.5\node_modules\impit\index.js:99:16)
        at Object.<anonymous> (C:\Codes\nodejs\tests\browser-test\node_modules\.pnpm\impit@0.2.5\node_modules\impit\index.js:335:17)
        at Module._compile (node:internal/modules/cjs/loader:1554:14) {
      code: 'ERR_DLOPEN_FAILED'
    }
  ]
}

Node.js v22.14.0
 ELIFECYCLE  Command failed with exit code 1.
```

cc @tocha688

--- a/impit-node/index.wrapper.js
+++ b/impit-node/index.wrapper.js
@@ -12,6 +12,10 @@ This can have several reasons:
         While the main package (impit) still installs, your package manager will skip installing the prebuilt native bindings for your platform.
         If you still want to skip installing other optional dependencies, please install the native bindings for your platform as a direct dependency of your project.
 - You are using a non-standard Node.js runtime (e.g. Deno, Bun, Cloudflare workers etc.) that might not support native modules.
+${
+    process.platform === 'win32' ? `- On Windows, sometimes the Visual C++ Redistributable for Visual Studio is missing.
+        Please install it from https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170` : ''
+}
 
 Run your script with IMPIT_VERBOSE=1 environment variable to get more information about the error.
 `, process.env['IMPIT_VERBOSE'] === '1' ? { cause: e } : undefined);

